src/fsharp/FSharp.Core/printf.fs

```diff
@@ -951,7 +948,7 @@ module internal PrintfImpl =
                 arr.[start + i] <- s.Pop()
             arr
         
-        member this.GetArgumentAndTypesAsArrays
+        member __.GetArgumentAndTypesAsArrays
```

Comment:
Obviously I'm pro __ as this is what many projects in the ecosystem use. (also suggested by FSharpLint)